lib/src/svg/parser_state.dart

```diff
@@ -1439,6 +1439,7 @@ class SvgParserState {
         defaultFillColor;
 
     if (explicitOpacity && color != null) {
+      opacity *= color.opacity;
```

Comment:
This doesn't seem right to me. Can you explain? Is there a test case SVG that is rendering incorrectly without this change but correctly with it?